packages/pkg-deps.json

```diff
@@ -2,8 +2,7 @@
    "debian" : {
       "build-requires" : [
          "debhelper",
-         "dh-python",
-         "dh-systemd"
+         "dh-python"
```

Comment:
wont that result in a missing build-dependency elsewhere?
surely 'dh-systemd' is not useless, it must be relevant to other releases, no?

Reply:
Hmm, yes it would. I see a workaround for Xenial in `packages/bddep` (which requires using the `--release` flag). I updated this PR to model after that approach. Adding support for release-specific definitions to `tools/read-dependencies` might be cleaner, especially if requirements continue to diverge. I'd rather not do that right now if the updated proposal is acceptable.